rule/rule.go

```diff
@@ -145,6 +145,9 @@ func LoadData(path, pkg string, data []byte) (*File, error) {
 		return nil, err
 	}
 	f := ScanAST(pkg, ast)
+	if err = checkFile(f); err != nil {
```

Comment:
Nit: `if err := checkFile(f); err != nil`, here and below.

Using `=` makes it look like there's a subtle reason not to shadow `err`, but since that's not the case, it's better to use `:=`.